server/modules/category/manager/categoryManager.js

```diff
@@ -65,6 +65,20 @@ const insertItem = async (user, surveyId, item, client = db) =>
     return itemDb
   })
 
+const insertItems = async (user, surveyId, items, client = db) =>
+  await client.tx(async t => {
+    await CategoryRepository.insertItems(surveyId, items, client)
+
+    await markSurveyDraft(surveyId, t)
+
+    const activities = items.map(item => ({
+        type: ActivityLog.type.categoryItemInsert,
+        params: item
+      })
+    )
+    await ActivityLog.logMany(user, surveyId, activities, client)
```

Comment:
Same as above, can these operations be performed concurrently?

Reply:
yes, sure

Reply:
my comment as before